lib/apiconfig/apiconfig.go

```diff
@@ -71,9 +72,9 @@ type KubeConfig struct {
 // TypeMetadata initialised to the current version.
 func NewCalicoAPIConfig() *CalicoAPIConfig {
 	return &CalicoAPIConfig{
-		TypeMetadata: unversioned.TypeMetadata{
-			Kind:       "calicoApiConfig",
-			APIVersion: unversioned.VersionCurrent,
+		TypeMeta: metav1.TypeMeta{
+			Kind:       KindCalicoAPIConfig ,
```

Comment:
Changed case of first letter.

Reply:
Yup - all our resources are now cap first letter